Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -81,18 +82,22 @@ class SimplifiedLoginViewController: UIViewController {
         view.addSubview(primaryButton)
         view.addSubview(linksView)
         view.addSubview(footerStackView)
-        setupConstraints()
         
         primaryButton.addTarget(self, action: #selector(SimplifiedLoginViewController.primaryButtonClicked), for: .touchUpInside)
         linksView.loginWithDifferentAccountButton.addTarget(self, action: #selector(SimplifiedLoginViewController.loginWithDifferentAccountClicked), for: .touchUpInside)
         linksView.continueWithoutLoginButton.addTarget(self, action: #selector(SimplifiedLoginViewController.continueWithoutLoginClicked), for: .touchUpInside)
         footerStackView.privacyURLButton.addTarget(self, action: #selector(SimplifiedLoginViewController.privacyPolicyClicked), for: .touchUpInside)
         
-        setupNavigationBar()
+        if isPhone {
+            setupConstraints()
+            setupNavigationBar()
+        } else {
+            setupiPadConstraints()
+        }
     }
     
     func setupNavigationBar(){
-        guard UIDevice.current.userInterfaceIdiom == .phone else {
+        guard isPhone else {
```

Comment:
Maybe we can remove this guard now? As we are checking for it on a higher level.

Reply:
Yeah, its twice now in case of forget and use this method, but can be removed